### DISCUSSION TITLE
Student benefits are not yet available even tho my request was approved

### DISCUSSION THREAD

**Discussion by MohamedOsamaa74** [community/community#108002] [GitHub Education]
Hi guys, so on 19 January i applied for a student pack and got approved with the message "Congrats! Your request for @MohamedOsamaa74 was approved and your benefits will be available soon."
Today is the 12th of February and the student benefits are not available yet.
I have a friend in the same faculty as me and they applied on January 26 and today the student benefits were available for them
Can anyone tell me why the student benefits are not available yet for me, it's getting really frustrating

**Comment by KarwanKhalid** [comment]
It sounds frustrating! Have you tried reaching out to the support team to ask about the delay? Sometimes there might be a hiccup in the process, and they could give you more info on what's going on. It's strange that your friend got their benefits already, but hopefully, they can sort it out for you soon.

**Reply by MohamedOsamaa74** [reply]
I tried but got nothing new..

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @MohamedOsamaa74 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!

**Reply by MohamedOsamaa74** [reply]
Thank you
